### ISSUE TITLE
Overlay info pages

### ISSUE THREAD

**Issue by rijsab** [jurialmunkey/skin.arctic.horizon#236]
When switching background to overlay the fanart is available on home screen and info pages. Is it possible to have an option to turn this off for home page and show only in info pages? In info pages this would make more sense as it will be better visible and leaves the home page 'clean' by using blurr mode.

**Comment by jurialmunkey** [comment]
So are you saying a setting where:
Base windows = Blur
Info dialogs = Overlay

Is that the idea? Or would *only* the home page use blur (i.e. library base window still uses overlay)?

**Comment by drinfernoo** [comment]
I actually prefer the approach from AZ2, where the info dialog is blurred, but the rest is overlay.

**Comment by rijsab** [comment]
@jurialmunkey 

Yes correct, it would make sense to see the overlay in info dialogue

**Comment by dberg81** [comment]
@jurialmunkey

Showing the fanart on the base windows looks busy when you have a screen full of widgets blocking the image, but using the fanart looks great as the background in the library views.

It would be great if we could set a static background image for the menu/widgets but see the fanart when browsing the libraries.  Is this possible, or would this involve a major overhaul to the skin?
